wolfssl/openssl/ssl.h

```diff
@@ -440,19 +444,17 @@ typedef STACK_OF(ACCESS_DESCRIPTION) AUTHORITY_INFO_ACCESS;
 #define X509_REQ_get_pubkey             wolfSSL_X509_get_pubkey
 #define X509_get_notBefore              wolfSSL_X509_get_notBefore
 #define X509_get0_notBefore             wolfSSL_X509_get_notBefore
-#define X509_getm_notBefore             wolfSSL_X509_get_notBefore
+#define X509_getm_notBefore             (ASN1_TIME*)wolfSSL_X509_get_notBefore
```

Comment:
wolfSSL_X509_get_notBefore() returns WOLFSSL_ASN1_TIME*.
ASN1_TIME is typedefed to WOLFSSL_AN1_TIME.
Why is this necessary?

Reply:
I'm not sure why I changed this. I've reverted it back to how it was.